core/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+
+from pure_pagination import EmptyPage
+from pure_pagination import Paginator
+from django.shortcuts import render
+
+
+def create_child_instance_from_parent(child_cls, parent_instance):
+    """
+    Create a child model instance from a parent instance
+    """
+    parent_cls = parent_instance.__class__
+    field = child_cls._meta.get_ancestor_link(parent_cls).column
+
+    child_instance = child_cls(**{
+        field: parent_instance.pk
+    })
+
+    child_instance.__dict__.update(parent_instance.__dict__)
+    child_instance.save()
+    return child_instance
+
+
+def render_paginate_list(request, object_list, template_name):
```

Comment:
That's just a shortcut used from several views, I went for refactoring its inside with django pure pagination instead of changing each view to not use it